cmd/maya-exporter/app/collector/zvol/metrics.go

```diff
@@ -68,6 +68,7 @@ type poolSyncMetrics struct {
 	zpoolLastSyncTime             *prometheus.GaugeVec
 	zpoolStateUnknown             *prometheus.GaugeVec
 	zpoolLastSyncTimeCommandError *prometheus.GaugeVec
+	zpoolListRequestRejectCounter prometheus.Gauge
```

Comment:
instead of using `prometheus.Gauge`,  using `promethus.Counter` will be better.

Reply:
Done